lib/elix/responders/gif_me.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Elix.Responders.GifMe do
+  @moduledoc """
+  Searches the Giphy public API and return the URL
+  of a GIF matching the search term.
+  """
+
+  use Hedwig.Responder
+
+  @base_url "http://api.giphy.com/v1/gifs/search"
+  @api_key "dc6zaTOxFJmzC" # Giphy’s public beta API key
+  @sample_size 10
+  @api_client Application.get_env(:elix, :api_client)
+
+  @usage """
+  gif me <term> - Replies with a GIF URL matching the term
+  """
+  hear ~r/gif me (.+)/i, msg do
```

Comment:
I posed this question to the hedwig issue tracker: https://github.com/hedwig-im/hedwig/issues/57